pandas/core/groupby/ops.py

```diff
@@ -221,13 +222,13 @@ def _get_cython_vals(self, values: np.ndarray) -> np.ndarray:
             values = ensure_float64(values)
 
         elif values.dtype.kind in ["i", "u"]:
-            if how in ["var", "prod", "mean"] or (
+            if how in ["var", "mean"] or (
```

Comment:
This is a micro optimization, but when hard coded I think it's better to use tuples than lists - they are stored as constants rather than constructed at runtime.

Reply:
Forgot to reply here. Would also do this after we are through. Some of them won't be necessary anymore I think